### ISSUE TITLE
fuse Docker container broken

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#3817]
## Steps to Reproduce the Problem

```sh
cd scripts/fuse/docker
./run_new_container.sh
```

## Expected Result

Working docker container.

## Actual Result

```sh
# kdb --version
KDB_VERSION: 0.9.4
SO_VERSION: 5
# ls mount/1/
ls: cannot access 'mount/1/user:': Invalid argument
ls: cannot access 'mount/1/system:': Invalid argument
ls: cannot access 'mount/1/dir:': Invalid argument
ls: cannot access 'mount/1/spec:': Invalid argument
ls: cannot access 'mount/1/cascading:': Invalid argument
ls: cannot access 'mount/1/proc:': Invalid argument
cascading:  dir:  proc:  spec:  system:  user:
```

## System Information

- Elektra Version: https://github.com/ElektraInitiative/libelektra/pull/3794

## Further Log Files and Output

In nohup.out there are many tracebacks like:

```
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 51, in starmapstar
    return list(itertools.starmap(args[0], args[1]))
  File "/root/elektra_fuse/mock_context.py", line 63, in _mock_process_context_and_run
    return func(*args, **kwargs)
  File "/root/elektra_fuse/elektra_fuse_interface.py", line 48, in getattr
    kdb_file_stat = _stat_kdb_file(path)
  File "/root/elektra_fuse/elektra_fuse_interface.py", line 106, in _stat_kdb_file
    resolved_file_path = get_kdb_file(os_path)
  File "/root/elektra_fuse/elektra_util.py", line 46, in get_kdb_file
    with _get_kdb_instance() as db:
  File "/root/elektra_fuse/elektra_util.py", line 34, in _get_kdb_instance
    kdb.goptsContract (contract, sys.argv, custom_envp, kdb.Key(), config)
AttributeError: module 'kdb' has no attribute 'goptsContract'
```

**Comment by dev2718** [comment]
I think this is due to a caching issue. Most likely an old (ancestor) build of the container (that did not yet include 'goptsContract') was cached on your system, yielding a mix of updated scripts with an old base.

Fix: delete all cached (ancestor) images.
Or as a quick workaround: temporarily add something like `RUN ls` at the top of the Dockerfile to invalidate the cache.

**Comment by markus2330** [comment]
Thank you, this fixed the issue. So I had a Docker container with the development version of Elektra 0.9.4 but not the released one? Or is actually 0.9.5 required?
